apps/webClient/package.json

```diff
@@ -16,7 +16,8 @@
     "react": "^16.10.0",
```

Comment:
@Robbiekruszynski  -- remove `package-lock.json` file, we only need `yarn-lock.json`

Reply:
just to double check, that should also include getting
rid of the node_modules && package.json within the webClient directory?

Reply:
we want the node_modules and package.json

`yarn` from the root of the repo will manage everything using `yarn workspaces` in the root package.json

Each application should have its own package.json and node_modules -- the node_modules will be linked to the root node_modules directory -- its just yarn monorepo magic, you don't need to know too many details yet except that you want to do `yarn add three` inside `apps/webClient` in order to add it to the webClient app